src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -50,8 +51,32 @@ class DetectedLanguageSuitev4 extends TestBase with DataFrameEquality with TextK
     assert(iso(0).get(0).toString == "en" && iso(0).get(1).toString == "es" &&
       iso(1).get(0).toString == "fr" && iso(1).get(1).toString == "zh")
   }
-}
 
+  test("Language Detection - Assert Confidence Score") {
+    val replies = detector.transform(df)
+      .select("output.result.name", "output.result.confidenceScore")
+      .collect()
+
+    val confidence = replies.map(row => row.getList(1))
+    assert(confidence(0).get(0).toString > "0.50")
+    assert(confidence(0).get(1).toString > "0.50")
+    assert(confidence(1).get(0).toString > "0.50")
+    assert(confidence(1).get(1).toString > "0.50")
+    assert(confidence(2).get(0).toString < "0.50")
+  }
+  test("Language Detection - Check Model Version") {
+    val outputCol = detector.transform(df)
+      .select("output")
+      .collect()
+    assert(outputCol(0).schema(0).name == "output")
+    val fromRow = DetectLanguageResponseV4.makeFromRowConverter
+    outputCol.foreach(row => {
+      val outResponse = fromRow(row.getAs[GenericRowWithSchema]("output"))
+      val modelCheck = outResponse.modelVersion.get == "2021-06-01"

```

Comment:
we want to check only the format. Can we remove "2021-06-01"? What does this line want to check?